cmd/aws-janitor-boskos/main.go

```diff
@@ -64,6 +70,10 @@ func main() {
 		sweepSleepDuration = d
 	}
 
+	if len(rTypes) == 0 {
+		logrus.Fatal("--resource-type must not be empty!")
```

Comment:
If you add a default for `aws-account` it will avoid needing to coordinate a kubernetes/test-infra PR to prevent from breaking the existing configuration for the aws janitor when this merges.

Reply:
Agree. Thank you.